### PR TITLE
Update N_0xd6ade981781fca09.md

### PR DESCRIPTION
**Pull request by crocodildos**

### CODE REVIEW

**Comment by technetium-cfx** [comment]
Could you please review the contribution guidelines and open a single PR for your changes? All of these are incorrect :/

File: MOBILE/CellCamSetHeadPitch.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+ns: MOBILE
+aliases: ["0xD6ADE981781FCA09"]
+---
+## _CELL_CAM_SET_HEAD_PITCH
+
+```c
+// 0xD6ADE981781FCA09
+void CELL_CAM_SET_HEAD_PITCH(float p0);
```
Comment (Disquse) [inline MOBILE/CellCamSetHeadPitch.md:9]: There should be `_` prefix too.

File: MOBILE/CellCamSetHeadPitch.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+```
```
Comment (stannum-cfx) [inline MOBILE/CellCamSetHeadPitch.md:12]: Please don't use code blocks for descriptions.

**Review by stannum-cfx — CHANGES_REQUESTED** [review]
As mentioned by Disquse, native name should be prefixed by `_`

**Pull request merged**